syntax = "proto3";
package custom;




import "google/protobuf/empty.proto";

/*
ReleaseIdcRackUnit请求
*/
message ReleaseIdcRackUnitRequest {
message DeviceList {
    /*
    实例ID
    */
    string instanceId = 1 ;
    /*
    模型ID
    */
    string objectId = 2 ;
}
        
    
    /*
    设备列表
    */
    repeated DeviceList deviceList = 1 ;
}

/*
ReleaseIdcRackUnitApi返回
*/
message ReleaseIdcRackUnitResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    google.protobuf.Empty data = 4 ;
}

